collect_app/src/androidTest/java/org/odk/collect/android/database/helpers/FormsDatabaseHelperTest.java

```diff
@@ -61,5 +64,17 @@ public void testMigration() throws IOException {
         List<String> newColumnNames = FormsDatabaseHelper.getFormsColumnNames(db);
 
         assertThat(newColumnNames, contains(FormsDatabaseHelper.CURRENT_VERSION_COLUMN_NAMES));
+
+        if (description.startsWith("Upgrading")) {
```

Comment:
How do you imagine such a check? I would need to open the database like I do here for example: https://github.com/opendatakit/collect/pull/3323/files#diff-0c930d81466089bfec86df65dbef3a4eR365 and compare versions but is it worth it?

Reply:
I just meant that in addition to the `description` and `dbFilename` parameters, it could have an `upgrading` parameter that gets set when the test is created. That way it's more self-documenting than the fact that the description has to have a particular name.

Reply:
ok done.